packages/babel/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-# @webpack-blocks/babel6 - Changelog
+# @webpack-blocks/babel - Changelog
+
+## Next version
+
+- Renamed `babel6` to `babel`
```

Comment:
I forgot to add a link to this PR here

Reply:
Plus: I would mention that it's the whole package that is renamed (wouldn't be sure on first glance otherwise).

`- Renamed package from 'babel6' to 'babel'`?

Reply:
Please also add that `babel-core` is now a peer dependency (breaking change!)